app/account/serializers.py

```diff
@@ -0,0 +1,30 @@
+import typing as tp
+
+from account.schemas import AccountData
+
+
+def account_serializer(model_objects: tp.Any,
+                       many: bool = False) -> tp.Tuple[
+    tp.Union[tp.List[tp.Dict[str, tp.Union[str, bool]]],
+             tp.Dict[str, tp.Union[str, bool]]], int
+]:
+    """
+    Serializer for account
+    Args:
+      model_objects: current Model
+      many: if many models
+    Returns:
+      dict with user data
+    """
+
+    schema = AccountData
+
+    if not many:
+        if model_objects is not None:
+            return schema(**model_objects.__dict__).__dict__, 200
+        return {}, 404
```

Comment:
serializer shouldn't be returning any http status codes, it's job is to turn stuff into json

Reply:
done